force-app/main/default/classes/GeocodingServiceTest.cls

```diff
@@ -29,15 +29,15 @@ private with sharing class GeocodingServiceTest {
         );
 
         // THEN
-        System.assert(
+        Assert.isTrue(
```

Comment:
These set of `Assert.isTrue` with equality checks should be turned into `Assert.areEqual`.